src/components/KPagination/KPagination.vue

```diff
@@ -114,17 +114,21 @@
         class="page-size-select"
         data-testid="page-size-dropdown"
       >
-        <KSelect
-          appearance="button"
-          :button-text="pageSizeText"
+        <KDropdown
           :items="pageSizeOptions"
           :kpop-attributes="kpopAttrs"
-          :placeholder="`${ currentPageSize } items per page`"
-          position-fixed
-          :test-mode="!!testMode || undefined"
-          width="190"
-          @selected="updatePageSize"
-        />
+          selection-menu
+          show-caret
```

Comment:
<img width="742" alt="Screenshot 2024-01-03 at 9 46 47 PM" src="https://github.com/Kong/kongponents/assets/2229946/ceb31c8f-0e0f-435b-aa57-4369595f283d">
question: should we add a max-height attribute to this pagination dropdown so that it's not super-tall and just scrolls internally?